### PR TITLE
Change object declaratation comma on multiline

### PR DESCRIPTION
**Pull request by nahody**

### CODE REVIEW

**Comment by lukas-zech-software** [comment]
:+1:  for the proposal

but please target the PR against *master* - the *develop* branch is not used in this repository and the diff of this PR is huge and missleading

File: es6.md [hunk 1]
```diff
@@ -104,31 +104,32 @@ const obj = {
...
 const a = {
   k1: v1,
   k2: v2,
 };
-const b = { k1: v1, k2: v2 };
+const b = { k1: v1, k2: v2, };
 const c = { k1: v2 };
 ```
 
 *Mostly pure convention. The trailing for multiline literals comma helps with adding/removing properties, and yield smaller and easier to read git diffs.*
```
Comment (Drexonom) [inline es6.md]: Remove this line, because it is getting obsolete with the new proposal.

**Comment by nahody** [comment]
@lukas-zech-software should i target it now or is it ok just for next time?

**Comment by nahody** [comment]
Sorry my mistake

**Comment by lukas-zech-software** [comment]
> @lukas-zech-software should i target it now or is it ok just for next time?

Its fine for me, just remember to merge develop to master after the PR is closed

**Comment by Drexonom** [comment]
Has the proposal been accepted? The merge result into master does not contain any changes of this proposal.

**Pull request merged**